lib/topological_inventory/persister/ar_helper.rb

```diff
@@ -0,0 +1,22 @@
+module TopologicalInventory
+  module Persister
+    module ArHelper
+      def self.load_environment!
+        require "active_record"
+
+        app_root = Pathname.new(__dir__).join("../../../").expand_path
+        $LOAD_PATH << app_root.join("app", "models")
+        $LOAD_PATH << app_root.join("app", "models", "concerns")
+
+        require "application_record"
+        Dir[File.expand_path("app/models/**/*.rb", app_root)].each { |f| require f }
```

Comment:
Don't need the `expand_path` here either `app_root.join("app/models/**/*.rb")`

Reply:
Done